modules/image-selector/outputs.tf

```diff
@@ -0,0 +1,30 @@
+########################################################################################################################
+# Outputs
+########################################################################################################################
+
+output "filtered_image_names_by_os" {
+  value       = [for img in local.os_images : img.name]
+  description = "List of image names available for the specified operating system."
+}
+
+output "filtered_image_names_by_architecture" {
+  value       = [for img in local.arch_images : img.name]
+  description = "List of image names matching the specified system architecture."
+}
+
+output "filtered_image_names_by_os_and_architecture" {
```

Comment:
no need to include `_by_os_and_architecture`

Reply:
Done, now changed to `output "filtered_image_names"`